IDEAS.org

```diff
@@ -31,14 +39,38 @@ Signed git tags should be highlighted.
   monitor key servers to make sure keys are up to date and modules signed with
   revoked keys have proper notifications for users
 
+***** Proposal(s):
+
++ To be discussed with some of their people of infrastructure and monitoring 
+  team. It involves setting up monitoring entries to those keyserver URL's and 
+  use this information to match the usage of this key it to the list of users
+  that downloaded/use the module. This can be achieved by expanding current
+  capabilities of the puppetlabs.com user accounts. Only registered users could
+  really benefit from this though. It also gives a better "popcon" statistics 
+  usage. 
++ It might be also implemented in the puppet server itself. like any linux 
+  distro's way of gathering software usage statistics (like debian's "popcon")
+  the puppet server could have a feature where modules downloaded directly from
+   the forge, send some anonymous statistics about the module usage.  
+  This can make use of the puppet server's administrator e-mail as identifier.
+  Enabling or disabling this feature could be something trivial as:
+  
+  "puppet resource forge_module_send-stats enable/disable" or something like 
+  that
+
 **** Avoiding hostile takeovers
 
+[fn:3] no ideas yet
+
 
 *** Different providers
 
  Not necessarily tied to Github, but could live on Gitlab, own Git installations
  etc.
 
++ +1 on this, but, it will increase complexity and workload for backend 
```

Comment:
I'm not sure about this -- as long as it's git, it shouldn't matter if it's hosted on Github or somewhere else? Unless I misunderstand you...